content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -75,9 +75,9 @@ A user creates, or in the case of dynamic provisioning, has already created, a `
 
 Claims will remain unbound indefinitely if a matching volume does not exist. Claims will be bound as matching volumes become available. For example, a cluster provisioned with many 50Gi PVs would not match a PVC requesting 100Gi. The PVC can be bound when a 100Gi PV is added to the cluster.
 
-### Using
+### Mounting and Using
 
-Pods use claims as volumes. The cluster inspects the claim to find the bound volume and mounts that volume for a Pod. For volumes that support multiple access modes, the user specifies which mode is desired when using their claim as a volume in a Pod.
+Pods use claims as volumes. The cluster inspects the claim to find the bound volume and mounts that volume for a Pod. Once the pvc which bound with pv has been used by pods, it will do actually volume mount on node. For volumes that support multiple access modes, the user specifies which mode is desired when using their claim as a volume in a Pod.
```

Comment:
@Colstuwjx this new wording doesn't seem to be idiomatic English. Can you revise it?

Reply:
Em.. I'm not native speaker, could you please give me some suggest changes?

Reply:
How about:
```suggestion
Pods use claims as volumes. The cluster inspects the claim to find a matching PersistentVolume for the Pod, and to check that the Pod is entitled to use it. If the claim is valid and the storage is available, the kubelet sets up storage access so that the PersistentVolume is available to containers in the Pod (either as mounted storage or as a block device). The PersistentVolumeClaim also defines whether the kubelet should mount the volume read-only or read-write.
```
```